server.js

```diff
@@ -1,13 +1,15 @@
 const Koa = require('koa');
 const Router = require('koa-router');
-const Logger = require('koa-logger');
+const koaLogger = require('koa-logger');
```

Comment:
Maybe `KoaLogger` (capital K)? I'm not sure. Take a decision about that.

Reply:
as convention variable names are in kababCase but for some reason many packages show his own library with capital letter. so i think both ways are fine